app/src/main/java/com/framgia/englishconversation/screen/dialog/OptionPostFragment.java

```diff
@@ -0,0 +1,66 @@
+package com.framgia.englishconversation.screen.dialog;
+
+import android.app.Dialog;
```

Comment:
ERROR: Unused import: android.app.Dialog. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports